client/packages/requisitions/src/RnRForms/DetailView/Footer.tsx

```diff
@@ -50,10 +67,7 @@ export const Footer = ({
             height={64}
           >
             <Box flex={1} display="flex" justifyContent="flex-end" gap={2}>
-              <DialogButton
-                onClick={() => navigateUpOne()}
-                variant={'cancel'}
-              />
+              <DialogButton onClick={() => navigateUpOne()} variant={'close'} />
```

Comment:
Seeing as this just navigates back to the list view, have changed the button to say close... Not sure whether we actually want it to be a cancel/discard changes button?

Reply:
Close sounds good to me. Could change it based on dirty state `close and discard`, but I think close works for now. Would be great to have this as a UI pattern, recorded somewhere